File: OmegaUp/8700. ASCII for first four characters.java
import java.util.Scanner;
public class Main {
  public static void main ( String[] args ) {
    Scanner scanner = new Scanner ( System.in );
    String name = scanner.next();
    for ( int i = 0; i < name.length(); i++ ) {
      System.out.printf ( "%c ASCII value is %d\n", name.charAt(i), (int) name.charAt(i) );
      if ( i == 3 ) break;
    }
  }
}

File: OmegaUp/8700. ASCII for first four characters.cpp
#include <iostream>
using namespace std;
int main () {
  string name;
  cin >> name;
  for ( int i = 0; i < name.length(); i++ ) {
    char c = name[i];
    printf ( "%c ASCII value is %d\n", c, (int)c );
    if ( i == 3 ) break;
  }
}
